internal/reminder/reminder.go

```diff
@@ -151,14 +173,19 @@ func (r *reminder) sendReminders(ctx context.Context) error {
 	repoIds := make([]uuid.UUID, len(repos))
 	for _, repo := range repos {
 		repoIds = append(repoIds, repo.ID)
+		if r.metrics != nil {
+			// sendDelay = Now() - ReminderLastSent - MinElapsed
+			reminderLastSent := repo.ReminderLastSent
+			if reminderLastSent.Valid {
+				r.metrics.SendDelay.Record(ctx, (time.Since(reminderLastSent.Time) - r.cfg.RecurrenceConfig.MinElapsed).Seconds())
```

Comment:
I'm still not sure about this metric. Let's say a repo has been frequently updated every hour (due to GH webhook events) and `r.cfg.RecurrenceConfig.MinElapsed` is 2 hours. In this case, let's say after sending a reminder 18 hours back, every hour the repo was updated by GH webhook and wasn't eligible for a reminder. Now when it becomes eligible for the reminder `SendDelay`  would be recorded as `18h` which seems to be very wrong in this case. (I think we discussed this previously too)